airflow/providers/amazon/aws/hooks/eventbridge.py

```diff
@@ -16,12 +16,76 @@
 # under the License.
 from __future__ import annotations
 
+import json
+
+from airflow import AirflowException
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+from airflow.providers.amazon.aws.utils import trim_none_values
+
+
+def validate_json(pattern: str) -> None:
+    try:
+        json.loads(pattern)
+    except ValueError:
+        raise AirflowException("Event pattern must be a valid JSON string.")
```

Comment:
I think `ValueError` makes more sense here

```suggestion
        raise ValueError("`event_pattern` must be a valid JSON string.")
```